pkg/provider/akeyless/akeyless_api.go

```diff
@@ -82,6 +84,7 @@ func (a *akeylessBase) GetToken(accessID, accType, accTypeParam string, k8sAuth
 
```

Comment:
The `RestAPI.Auth(ctx)` in line `:70` is not being observed, i suggest to add a `ObserveAPICall()` there, too